### DISCUSSION TITLE
Slot Recipe Select

### DISCUSSION THREAD

**Discussion by OmarDronodat** [chakra-ui/chakra-ui#9349] [Q&A] [answered]
I am working on select 
![image](https://github.com/user-attachments/assets/c156eb46-37f9-4287-88c1-2bf52b7a264b)

and I am testing slotrecipe on it to try to change the color of the background but i am finding difficulty for some reason
![image](https://github.com/user-attachments/assets/c27809cc-3e4e-4fef-8b6c-5f9f687f97bf)
I searched and i thought that choosing trigger will help me do it but unfortunaltly no avail
for reference i want to change the green area
![image](https://github.com/user-attachments/assets/62099854-c6aa-4ced-a002-db8ebf2a50e8)

**Comment by segunadebayo** [comment, marked as answer]
The green area is the `control` slot, not `trigger`. Include the anatomy slots or the recipe never attaches.

```ts
defineSlotRecipe({
  slots: selectAnatomy.keys(),
  base: { control: { bg: "white" } },
})
```
